notebook/notebookapp.py

```diff
@@ -1100,6 +1109,15 @@ def start(self):
         
         This method takes no arguments so all configuration and initialization
         must be done prior to calling this method."""
+        try:
+            is_root = os.geteuid() == 0
+        except AttributeError as e:
+            import ctypes
+            is_root = ctypes.windll.shell32.IsUserAnAdmin() == 1
```

Comment:
I wouldn't do this - admin users in Windows aren't really equivalent to root on Unix, and I don't think it makes sense to block them from running the notebook. Admin users on Windows are more like the sudoers group on Linux.